package.json

```diff
@@ -11,7 +11,9 @@
     "test": "test"
   },
   "scripts": {
-    "test": "mocha"
+    "test": "mocha",
+    "lint": "eslint .",
+    "posttest": "npm run lint"
```

Comment:
`posttest` will run the linter automatically at the end of `npm test`. This will mean that Travis should run the linter automatically once it's hooked up.